src/main/java/ladder/domain/ladder/Stairs.java

```diff
@@ -0,0 +1,54 @@
+package ladder.domain.ladder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Stairs {
+
+    private static final int EXCEPT_FIRST_AND_LAST_PILLAR_COUNT = 2;
+
+    private final List<Stair> stairs;
+
+    private Stairs(final List<Stair> stairs) {
+        this.stairs = stairs;
+    }
+
+    public static Stairs of(final PillarCount pillarCount) {
+        return new Stairs(createHorizontalStairs(pillarCount));
+    }
+
+    private static List<Stair> createHorizontalStairs(final PillarCount pillarCount) {
+        List<Stair> stairs = new ArrayList<>();
+
+        Stair currentStair = createFirst();
+        stairs.add(currentStair);
+        stairs.addAll(createMiddle(currentStair, pillarCount.getValue() - EXCEPT_FIRST_AND_LAST_PILLAR_COUNT));
+        stairs.add(createLast(currentStair));
+
+        return stairs;
+    }
+
+    private static Stair createFirst() {
+        return Stair.createOfFirstPillar();
+    }
+
+    private static List<Stair> createMiddle(final Stair preStair, final int middlePillarCount) {
+        List<Stair> middleStairs = new ArrayList<>();
+        Stair stair = preStair;
+
+        for (int i = 0; i < middlePillarCount; i++) {
+            stair = stair.createOfMiddlePillar();
+            middleStairs.add(stair);
+        }
+
+        return middleStairs;
+    }
+
```

Comment:
List<Stair> stairs 필드를 생성자에서 초기화 하면, prevStair 파라미터를 없앨수 있을거 같아요
```suggestion
    private static void createMiddle(final int middlePillarCount) {
        if (middilePillarCount < 0) {
            return;
        }
        
        Stair stair = preStair().createOfMiddlePillar();
        stairs.add(stair);
        createMiddle(middlePillarCount - 1);
    }
    
    private Stair prevStair() {
        return stairs.get(stairs.size() - 1);
    }

```

Reply:
https://github.com/next-step/java-ladder/pull/403#discussion_r434902650
- 이 부분에서 이런 피드백을 받아 우선 불필요한 메서드 분리를 제거하고 하나의 메서드에서 `createHorizontalStairs()` 의 행위를 표현하도록 수정하였습니다.
- 이로 인해 `createMiddle()`에서 prevStair 인자를 받는 부분이 제거되어 이 부분에 대한 피드백은 다음 과정에서 추가로 진행해주셨으면 좋겠습니다~ :)